VrpInstance.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from typing import Optional
+
+import numpy as np
+import numpy.typing as npt
+
+
```

Comment:
We have four types of instances:
- Epoch instances
- Dispatch instances
- Hindsight instances
- Scenario instances